### PR TITLE
Issue #3217219 by vnech: Set correct langcode when create a content in other then default language first

### PR DESCRIPTION
**Pull request by nechai**
## Problem
There is an issue with the "langcode" field when creating entities in multilanguage OS.

For example, I have two languages installed - "English" (default) and "Ukrainian".
If I create a Topic in "Ukrainian" first the wrong "langcode" sets in the node (the field has "en" value instead of "uk")

## Solution
Set correct "langcode" on form alter for new entities.

## Issue tracker
- https://www.drupal.org/project/social/issues/3217219
- https://getopensocial.atlassian.net/browse/YANG-5782

## How to test
*For example*
- [ ] Login as admin
- [ ] Install additional language ("French", for example)
- [ ] Switch to "Franch" language and create a Topic
- [ ] Go to created topic translations page

## Release notes
Set correct langcode when creating content in different than default language

### CODE REVIEW

File: modules/custom/social_language/social_language.module [hunk 1]
```diff
@@ -48,3 +50,26 @@ function social_language_social_group_default_route_tabs_alter(&$tabs) {
   // Unset some tabs created by group.
   unset($tabs['content_translation.local_tasks:entity.group.content_translation_overview']);
 }
+
+/**
+ * Implements hook_form_alter().
+ */
+function social_language_form_alter(&$form, FormStateInterface $form_state) {
```
Comment (ribel) [inline modules/custom/social_language/social_language.module:57]: @nechai I think it makes more sense to move this code to `social_content_translation.module`.
Also, did you investigate why it doesn't work correct by default? 
Are there some drupal.org issues related to that?

**Review by ribel — CHANGES_REQUESTED** [review]

File: modules/custom/social_language/social_language.module (hunk 1 above)
Comment (nechai) [reply modules/custom/social_language/social_language.module]: @ribel I moved the hook to the "social_content_translation" module. 
This works by default in clear Drupal but in OS field "langcode" is hidden in entities form displays (with default value "en").

**Comment by Kingdutch** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2370-xhuhwyfhlz79njfvcrwpxkywijsouw3b.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60c875c77571464ecda59c6e

File: modules/custom/social_language/social_language.module [hunk 2]
```diff
@@ -5,7 +5,9 @@
  * Contains social_language.module.
  */
 
+use Drupal\Core\Entity\ContentEntityInterface;
```
Comment (ribel) [inline modules/custom/social_language/social_language.module:8]: @nechai I think those use should be removed?

Comment (nechai) [reply modules/custom/social_language/social_language.module:8]: @ribel This class is used below...

Comment (nechai) [reply modules/custom/social_language/social_language.module]: @ribel Removed unused class names

**Comment by Kingdutch** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr2370-xhuhwyfhlz79njfvcrwpxkywijsouw3b.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60c875c77571464ecda59c6e

**Pull request merged**